JPS_Ontology/ontology/ontobuildingstructure/OntoBuildingStructure.csv

```diff
@@ -1,47 +1,46 @@
+RoofArea,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Area,IS-A,,,,The size of the roof on a building,https://www.theworldavatar.com/kg/ontobuildingstructure/,Roof Area
+SolarSuitableRoofArea,Class,RoofArea,IS-A,,,,The size of the roof area of a building that is suitable for solar use.,https://www.theworldavatar.com/kg/ontobuildingstructure/,Solar Suitable Roof Area
+TotalRoofArea ,Class,RoofArea,IS-A,,,,The total size of the roof of a building.,https://www.theworldavatar.com/kg/ontobuildingstructure/,Total Roof Area 
+hasQuantity ,"Object Property, Asymmetric Property",,,http://www.w3.org/2002/07/owl#Thing,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,,"Links any entity to a specific quantity defined in the OM ontology. This relationship is not intended to be stated explicitly, but inferred from its sub-properties. For application, only the sub-properties are used.",https://www.theworldavatar.com/kg/ontobuildingstructure/,has quantity
+hasArea,"Object Property, Asymmetric Property",hasQuantity,IS-A,Roof,RoofArea,,A relationship between the roof of a building and the area it covers.,https://www.theworldavatar.com/kg/ontobuildingstructure/,has area of
+hasOrientation,"Object Property, Asymmetric Property",hasQuantity,IS-A,InclinedRoof,http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,,A relationship between a roof and its angle/position relative to the Sun.,https://www.theworldavatar.com/kg/ontobuildingstructure/,has orientation
+hasSlope ,"Object Property, Asymmetric Property",hasQuantity,IS-A,InclinedRoof,http://www.ontology-of-units-of-measure.org/resource/om-2/Angle,,A relationship between a roof and its slope angle.,https://www.theworldavatar.com/kg/ontobuildingstructure/,has slope
+hasFacade,Object Property,,,https://w3id.org/bot#Building,RoofFacade UNION WallFacade,,A relation that links a building to their roof and wall facades.,https://www.theworldavatar.com/kg/ontobuildingstructure/,has fa�ade
+compriseOf,Object Property,,,RoofFacade,Roof,only,A relation that denotes the subcomposition a roof or wall fa�ade and their individual roofs or walls. ,https://www.theworldavatar.com/kg/ontobuildingstructure/,comprise of
+compriseOf,Object Property,,,WallFacade,Wall,only,A relation that denotes the subcomposition a roof or wall fa�ade and their individual roofs or walls. ,https://www.theworldavatar.com/kg/ontobuildingstructure/,comprise of
+consistsOf,"Object Property, Asymmetric Property",,,Stair,Landing UNION Railing UNION StairFlight UNION StairStructuralComponent,only,"A relationship that denotes the subcomposition of an assembly with their individual components. An example is that stairs consists of railing, landing, stair flights, and stair structural components.",https://www.theworldavatar.com/kg/ontobuildingstructure/,consists of
```

Comment:
Similarly to `comprisesOf`, you need an extra line that defines the domain and range as the unions of the domains and ranges of the two separate definitions.